internal/checks/source-code/contribution-access/contribution_access_test.go

```diff
@@ -3,261 +3,94 @@ package contributionaccess
 import (
 	_ "embed"
 	"testing"
-	"time"
 
 	"github.com/aquasecurity/chain-bench/internal/checks/common"
 	"github.com/aquasecurity/chain-bench/internal/models/checkmodels"
 	"github.com/aquasecurity/chain-bench/internal/testutils"
 	"github.com/aquasecurity/chain-bench/internal/testutils/builders"
-	"github.com/aquasecurity/chain-bench/internal/utils"
 )
 
 func TestOrganizationChecker(t *testing.T) {
 	tests := []testutils.CheckTest{
 		{
-			Name: "Organization with unverified status, and missing permission to fetch org and repo settings",
+			Name: "Organization without org settings permissions, without admin permissions, and with no commits ",
```

Comment:
What should happen? should fail? with what reason?
the test name here (and for all of the tests) describes the situation, and not what should actually happen

Tests should be treated also like the documentation of the app behavior.